autonomous/autonomous.py

```diff
@@ -13,12 +13,32 @@
 from controllers.shooter import ShooterController
 from controllers.indexer import IndexerController
 from utilities import trajectory_generator
+from typing import List
+from enum import Enum
 
 
-class AutoBase(AutonomousStateMachine):
-    """Follows a smoothed linear path between waypoints
-    if it has a ball it stops before the next waypoint to fire it, otherwise dosent stop"""
+class WaypointType(Enum):
+    PICKUP = 0
+    SHOOT = 1
+    WAYPOINT = 2
```

Comment:
nit: presumably these values do not matter, we can use `enum.auto()` in that case